<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>916</width>
    <height>617</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Simulation segmentation</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="1">
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="buttonReload">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>reload</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="buttonPlay">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Play</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="buttonStop">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Stop</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="labelInformations">
          <property name="sizePolicy">
           <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Informations :</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item row="0" column="1">
     <widget class="PictureWidget" name="pictureWidget" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </item>
    <item row="0" column="0" rowspan="2">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QPushButton" name="buttonImageLoading">
        <property name="text">
         <string>Charger Image</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="buttonVideoLoading">
        <property name="text">
         <string>Charger Vid√©o</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="buttonForegroundDrawing">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Rouge</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="buttonBackgroundDrawing">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Bleu</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="buttonClear">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Effacer</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="buttonTreatment">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Traitement</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="buttonSave">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Enregistrer</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="2" column="1">
     <widget class="QSlider" name="horizontalSlider">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>916</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PictureWidget</class>
   <extends>QWidget</extends>
   <header>picturewidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
